pkg/operator/operator.go

```diff
@@ -7,6 +7,7 @@ import (
 	"database/sql"
 	"fmt"
 	"io/ioutil"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
minor nit: I would move this to the kube-related imports. We're pretty consistent about this throughout the codebase.

Reply:
Fixed in 59e02b58